Resources/Maps/Shuttles/DeltaV/NTES_Lox.yml

```diff
@@ -2804,11 +2794,15 @@ entities:
   - uid: 555
     components:
     - type: Transform
+      anchored: False
```

Comment:
Looks like the Lox has a duplicate pipe here

Reply:
Thats nothing I had to do with, but ill hunt it down

Reply:
I know, just figure while you're in here...thanks 😁